docs/pages/technology/did.rst

```diff
@@ -0,0 +1,53 @@
+.. _did:
```

Comment:
I feel like this is something that should be part of the nuts-specification? Maybe not of a specific RFC but e.g. "intro to DIDs and Nuts" or something. Because it really applies to DID/Nuts spec in general, not our specific ref implementation.

Reply:
Specs should explain anything that is covered in other specs. But if you start with the node, you may want to know what it's doing and why. When using tech, it can be annoying to refer to specs.